client-spark/src/test/java/utils/EnvReader.java

```diff
@@ -0,0 +1,40 @@
+package utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Properties;
+import models.S3TestConfig;
+
+public class EnvReader {
+
+  public S3TestConfig readS3TestConfig() {
+    Properties properties = readProperties();
+    String region = properties.getProperty("WHITEFOX_TEST_AWS_REGION");
+    String accessKey = properties.getProperty("WHITEFOX_TEST_AWS_ACCESS_KEY_ID");
+    String secretKey = properties.getProperty("WHITEFOX_TEST_AWS_SECRET_ACCESS_KEY");
+    return new S3TestConfig(region, accessKey, secretKey);
+  }
+
+  private Properties readProperties() {
```

Comment:
this should be automatically done by the env plugin, you just need to set it up in the gradle build definition